README.md

```diff
@@ -1,2 +1,3 @@
 # git-demo
 to demonstrate how to use git and GitHub
+- hongtong linhttps://github.com/hongtonglin
```

Comment:
Could you make your name into a link that goes to that URL? The URL should not be visible!